modules/assim.batch/tests/Rcheck_reference.log

```diff
@@ -140,9 +71,32 @@ File ‘PEcAn.assim.batch/R/pda.get.model.output.R’:
  OK
 * checking for unstated dependencies in vignettes ... OK
 * checking package vignettes in ‘inst/doc’ ... OK
-* checking re-building of vignette outputs ... OK
+* checking re-building of vignette outputs ... WARNING
```

Comment:
I understand that this gets us around the isFalse issue, but I don't feel great about switching a reference log from an OK to a WARNING (the idea of the log was to only move in the direction towards fewer warnings)